pkg/plugin/v3/scaffolds/internal/templates/componentconfig.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templates
+
+import (
+	"sigs.k8s.io/kubebuilder/v2/pkg/model/file"
+)
+
+var _ file.Template = &ComponentConfig{}
+
+// ComponentConfig scaffolds the ComponentConfig file in manager folder.
+type ComponentConfig struct {
```

Comment:
What about
```suggestion
type ManagerConfig struct {
```
or
```suggestion
type ControllerManagerConfig struct {
```
?
Apply the corresponding change also to this file name (`[controller_]manager_config.go`) and to the `f.Path` field  (`[controller_]manager_config.yaml`) too.

If you opt for the "controller" prefixed version, probably the patch should also have the same prefix to share the same scheme.

Reply:
Yeah, we also probably want to rename https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugin/v3/scaffolds/internal/templates/config/manager/config.go at some point too.